src/js/components/header.tsx

```diff
@@ -17,7 +17,7 @@ const Header = () => {
 
   const controls = () => (
     <PageHeaderControl className="slds-grid slds-grid_vertical-align-center">
-      {window.GLOBALS.ENABLE_WALKTHROUGHS ? <TourDropdown /> : null}
+      {/* {window.GLOBALS.ENABLE_WALKTHROUGHS ? <TourDropdown /> : null} */}
```

Comment:
Instead of passing `user` as a prop to the dropdown, we can have the `TourDropdown` component get `User` from the global state. See: https://github.com/oddbird/Metecho/commit/5cc2f5bd12f0d6f7b8b51d1ebfb25b332bf9455d